### DISCUSSION TITLE
How to see if Coral USB is being used

### DISCUSSION THREAD

**Discussion by wpeterw** [blakeblackshear/frigate#474] [Ask A Question]
I've installed and configured my Coral USB. Is there a way to see (logging ?) if the Coral is beeing used ? I see interference speeds of 200ms+.

**Comment by mattheys** [comment]
Likely your Coral is not being used if the inference speed is 200ms, it should probably be less than 60ms even on a slow machine and 20ms on a good one.

Post your sanitised docker command or compose file, I didn't have to install any Coral stuff on the host os, everything that was needed was in the container.

**Reply by wpeterw** [reply]
You are right:

`frigate    |  * Serving Flask app "detect_objects" (lazy loading)
frigate    |  * Environment: development
frigate    |  * Debug mode: off
frigate    | No EdgeTPU detected. Falling back to CPU.`

Now to find out why, my Coral USB shows up as:

`Bus 002 Device 004: ID 1a6e:089a Global Unichip Corp.` So, not as a Google device.

**Reply by mattheys** [reply]
There is an issue #381 where it is not found on boot and restarting the container fixes it. You could try that.

**Reply by wpeterw** [reply]
I got it working now, I've noticed the usb path changes when unpluggin/pluggin in the device.

**Reply by TheSleepySlee** [reply]
On USB coral, the white led blinks when it's in use. Easy way to check things out.
